coverage/report.md

```diff
@@ -205,6 +205,13 @@
 | `udat_toPatternRelativeDate` | |
 | `udat_toPatternRelativeTime` | |
 | `udat_unregisterOpener` | |
+Warning: Implemented + Unimplemented != Total
```

Comment:
?

Reply:
Is the question about the clarity of the message or the existence of the message? If it is about clarity, would something like "Warning: Count of total C API functions is not equal to the sum of the counts of implemented and non-implemented functions" be clearer?

Reply:
An alternative would be to add a screen to the '*_implemented.txt' file creation to ignore functions that aren't in the C API for the given header file.